<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_main</class>
 <widget class="QMainWindow" name="cell_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QMainWindow {
    background-color: #f0f2f5; 
}*/
  QMainWindow#cell_main &gt; QWidget#centralwidget { 
    background-color: #f0f2f5; 
  }

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>18</width>
    <height>18</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="Tool" native="true">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#Tool {
    background-color: #fff;
    border-radius: 8px;
    padding: 10px; 
    border: 1px solid #e5e5e5;
}

/* 按钮默认样式 */
QPushButton {
    border: none; /* 去掉默认边框 */
    border-radius: 0; /* 无圆角，符合需求 */
    padding: 10px 16px; /* 按钮内边距，让按钮更美观 */
    font-size: 14px; /* 字体大小 */
    background-color: #ECECEC; /* 默认背景色 */
    color: #333; /* 默认文字颜色 */
    border-bottom: 1px solid #CCC; /* 按钮之间的下边框，做分隔用 */
}

/* 按钮鼠标悬浮样式（可选，增强交互反馈） */
QPushButton:hover {
    background-color: #DDD; 
}

/* 按钮选中（点击后保持）样式 */

QPushButton:checked {
    background-color: #333; 
    color: #FFF; 
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="BorrowingRecordButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>借阅记录</string>
         </property>
         <property name="icon">
          <iconset resource="../qrc.qrc">
           <normaloff>:/new/prefix1/qrcpicture/historymanager.png</normaloff>:/new/prefix1/qrcpicture/historymanager.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="BookManagementButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>图书管理</string>
         </property>
         <property name="icon">
          <iconset resource="../qrc.qrc">
           <normaloff>:/new/prefix1/qrcpicture/bookmanager.png</normaloff>:/new/prefix1/qrcpicture/bookmanager.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="UserManagementButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>用户管理</string>
         </property>
         <property name="icon">
          <iconset resource="../qrc.qrc">
           <normaloff>:/new/prefix1/qrcpicture/usermanager.png</normaloff>:/new/prefix1/qrcpicture/usermanager.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
